content/en/docs/reference/config/analysis/ist0102/index.md

```diff
@@ -5,9 +5,9 @@ owner: istio/wg-user-experience-maintainers
 test: no
 ---
 
-This message occurs when you have a namespace that is missing the
-[annotation](/docs/reference/config/annotations/) to indicate whether the
-namespace is auto-injected, for example `sidecar.istio.io/inject`.
+This message occurs when a namespace is missing the `istio-injection` and `istio.io/rev`
```

Comment:
This is not `and`, as if we have one of the labels, it will be fine.

Reply:
Maybe something like:

This message occurs when a namespace is missing either the `istio-injection` label (which enables/disables sidecar injection) or the `istio.io/rev` label (which specifies the Istio control plane revision for the sidecar), or when the .values.sidecarInjectorWebhook.enableNamespacesByDefault is not enabled (which indicates if namespaces should be auto-injected with the sidecar by default).